packages/hardhat-core/src/types/config.ts

```diff
@@ -134,15 +134,15 @@ export type HardhatNetworkChainsConfig = Map<
 export interface HardhatNetworkConfig {
   chainId: number;
   from?: string;
-  gas: "auto" | bigint;
-  gasPrice: "auto" | bigint;
+  gas: "auto" | number;
+  gasPrice: "auto" | number;
   gasMultiplier: number;
-  initialBaseFeePerGas?: bigint;
+  initialBaseFeePerGas?: number;
   hardfork: string;
   mining: HardhatNetworkMiningConfig;
   accounts: HardhatNetworkAccountsConfig;
-  blockGasLimit: bigint;
-  minGasPrice: bigint;
+  blockGasLimit: number;
+  minGasPrice: number;
```

Comment:
this one used to be a BN, so I'd change it to bigint

Reply:
Changed in 236fb02c0b49e39990a714fe5d3f447e049ab722